src/SourceBuild/content/repo-projects/efcore.proj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <ItemGroup>
+    <RepositoryReference Include="arcade" />
+    <RepositoryReference Include="runtime" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(DotNetBuildSourceOnly)' == 'true'">
+    <RepositoryReference Include="source-build-reference-packages" />
+  </ItemGroup>
```

Comment:
is efcore supposed to be included in source-build or not? in the latter case I don't understand why we'd need this

Reply:
Agreed. I took the liberty of pushing an update.